### ISSUE TITLE
systemd-logind.service fails to print during startup phase because /var/lib is read-only for a period of time

### ISSUE THREAD

**Issue by cryingworld** [systemd/systemd#28975]
### Component

systemd-logind

### Is your feature request related to a problem? Please describe

During the startup phase of the system, I found that the systemd-logind.service startup printed multiple failures, but after the startup was completed, I found that the systemd-logind.service started successfully because for a period of time in the startup, the permissions of /var/lib were read-only, which led to the systemd- logind.service prints startup failure several times.

### Describe the solution you'd like

I think we should add a statement to determine the read/write permissions of /var/lib.

[Unit]
Description=User Login Management
Documentation=man:sd-login(3)
Documentation=man:systemd-logind.service(8)
Documentation=man:logind.conf(5)
Documentation=man:org.freedesktop.login1(5)

ConditionPathIsReadWrite=/var/lib

Wants=user.slice modprobe@drm.service
After=nss-user-lookup.target user.slice modprobe@drm.service

# Ask for the dbus socket.
Wants=dbus.socket
After=dbus.socket

**Comment by poettering** [comment]
I am sorry, what? /var/lib was read-only? how can that happen?

Generally, the assumption is that once local-fs.target is completed /var/ is writable. logind is started after that.

Why was /var/lib/ read-only for you? 

Please elaborate?

**Comment by cryingworld** [comment]
I turned on selinux, and local.fs.target's volatile-binds related services mounted it as an overlayfs, and under selinux, /var/lib is a read-only overlayfs filesystem. Unfortunately, I found that the audit logs did not give the relevant alerts, so I shut down the volatile-binds service, and in several tests I found that there was a read-only situation with /var/lib for a period of time during the first startup of the system, but after basic.target, the systemd-logind. service started successfully and did not occur during the subsequent multiple reboot tests. Now I just think it might be better to add this /var/lib path read/write permission determination
